tests/ci/s3_helper.py

```diff
@@ -47,7 +48,7 @@ class S3Helper:
 
     def __init__(self):
         config = botocore.config.Config(max_pool_connections=self.max_pool_size)
-        self.session = boto3.session.Session(region_name="us-east-1")
+        self.session = boto3.session.Session(region_name=S3_REGION)
```

Comment:
`us-east-1` is seen in many other places, as well as in new once. Let's use `S3_REGION` in get_robot_token.py and jepsen_check.py too